### ISSUE TITLE
Share trained ResNet-50 model in IN-9

### ISSUE THREAD

**Issue by tataiani** [MadryLab/backgrounds_challenge#3]
Is it possible to share your trained ResNet-50 model in IN-9? We've trained a ResNet-50 model from scratch with IN-9 and got ~96% accuracy, which matches your result. But, the same model showed more than 99% of the foregrounds vulnerable when tested with your challenge evaluation code and data. Hence I wanted to check with your trained model and find out the source of error. Thanks in advance.

**Comment by kaixiao** [comment]
Hi - we just added a link to download the ResNet-50 model we trained on IN-9L in the README. Here is the full link as well: https://www.dropbox.com/s/5bmfiunmlh0hy8n/in9l_resnet50.pt?dl=0

Just to clarify: This model is trained on IN-9L, not IN-9. IN-9L is a larger version of the IN-9 dataset. This model should have 96.4% accuracy and has 88% foregrounds vulnerable when run with the challenge evaluation code and data.
